Configuration/webapp/package.json

```diff
@@ -18,6 +18,9 @@
     "@aliceo2/web-ui": "^2.7.4",
     "@emotion/react": "^11.14.0",
     "@emotion/styled": "^11.14.0",
+    "@fortawesome/fontawesome-svg-core": "^6.7.2",
```

Comment:
We can leave it for now so that we do not block the development, but we agreed on using the material ui framework which also includes icons.
In the future, please reduce the number of dependencies unless critical. This is to avoid issues such as npm supply chain attacks which are more and more frequent